toc/working-groups/foundational-infrastructure.md

```diff
@@ -368,6 +368,7 @@ areas:
   - cloudfoundry/bosh-azure-storage-cli
   - cloudfoundry/bosh-apt-resources
   - cloudfoundry/bosh-bbl-ci-envs
+  - cloudfoundry/bosh-blobstore-clis
```

Comment:
In the mean time there is an RFC about this topic. The RFC suggests to add a new are called "Storage CLI" in https://github.com/cloudfoundry/community/blob/8fc25e9393de1624c84c77f71b2786134b35c852/toc/rfc/rfc-draft-cc-blobstore-storage-cli.md?plain=1#L39-L48. We should follow the RFC and create a new area for it combining the suggested approvers from CAPI.